tools/mayan_sync_helper/mayan_sync_transformer.py

```diff
@@ -0,0 +1,220 @@
+
```

Comment:
please either add some doc for this, or update the make file.

Reply:
Thanks for adding this!

Reply:
good idea. I will add a new pr with a readme